docs/upgrade_operators.md

```diff
@@ -0,0 +1,96 @@
+# Upgrade database operators
+
+Starting with Percona Everest 1.0.0, the intuitive UI allows you to upgrade database operators and all their components in any namespace with a single-click.
+
+## Before you upgrade
+
+Before upgrading a database operator, Everest presents a list of tasks that need to be completed to smoothly transition your clusters to the next version of the database operators. These tasks may include:
+
+- Upgrading your database engine version from an unsupported or end-of-life (EOL) version.
+
+- Ensuring your clusters are using a supported [Custom Resource](https://ibm.github.io/kubernetes-operators/lab1/#:~:text=A%20CRD%20defines%20Custom%20Resources,store%20and%20retrieve%20structured%20data.){:target="_blank"} (CR) version.
+
+
+- Backing up your databases.
```

Comment:
Do we have this at all? I don't think we do. We can make a suggestion as a note that we recommend that users take a backup before starting the upgrade procedure but saying that this is one of the tasks that we'll ask them to do before starting the upgrade is incorrect I believe.

Reply:
Fixed.